src/mongo/s/commands/cluster_count_cmd.cpp

```diff
@@ -184,11 +172,12 @@ class ClusterCountCmd : public ErrmsgCommandDeprecated {
                 resolvedView.asExpandedViewAggregation(aggRequestOnView.getValue());
             auto resolvedAggCmd = resolvedAggRequest.serializeToCommandObj().toBson();
 
-            BSONObj aggResult = Command::runCommandDirectly(
-                opCtx, OpMsgRequest::fromDBAndBody(dbname, std::move(resolvedAggCmd)));
+            BSONObjBuilder aggResult;
+            Command::findCommand("aggregate")
```

Comment:
Why did this line change? Is this just the result of merging with HEAD? Looks like a bad merge, as I think we should continue to use `Command::runCommandDirectly()`.